package.json

```diff
@@ -88,9 +90,9 @@
     "rollup-plugin-terser": "^7.0.2",
     "rollup-plugin-unassert": "^0.5.0",
     "selenium-webdriver": "^4.4.0",
+    "serve-static": "^1.15.0",
     "shuffle-seed": "^1.1.6",
     "sinon": "^15.0.0",
-    "st": "^3.0.0",
```

Comment:
`st` is still used in `start-server` script and consequently other scripts like `start` — let's either keep it, or replace the static server in the script with `serve-static` (CLI equivalent with same settings).

Reply:
There is no CLI for `serve-static,` so I'll keep `st`